### DISCUSSION TITLE
Can see protected routes without entering password

### DISCUSSION THREAD

**Discussion by jinoo2005609** [spencerwooo/onedrive-vercel-index#424] [General]
First time, When I try to view protected folders, it shows "Enter Password".
But when I enter the correct password and logout, then anyone can see the protected folders without entering password.

You can test in [Spencer's OneDrive](https://drive.spencerwoo.com/%F0%9F%8C%9E%20Private%20folder/u-need-a-password/) or your own vercel index.
(Maybe you can see this folder without entering password cuz I already entered password once.)

**Comment by spencerwooo** [comment]
Nope, I don't think I can reproduce this.

**Reply by opoet** [reply]
I reproduced it, and I will provide the video later.

**Reply by spencerwooo** [reply]
Oh, never mind, I reproduced it. Might be related to the caching mechanism.

**Comment by spencerwooo** [comment]
Yes, this is related to the caching mechanism. I disabled caching for password protected routes, check the latest commit and see if this solves the issue.

**Reply by jinoo2005609** [reply]
Oh, I merged latest commit and now it's fixed I think.
I really appreciate your help in resolving the problem.
